lib/MAST/Ops.nqp

```diff
@@ -4784,81 +4819,88 @@ BEGIN {
     'speshguardgetattr', 801,
     'atomicbindattr_o', 802,
     'casattr_o', 803,
-    'sp_guard', 804,
-    'sp_guardconc', 805,
-    'sp_guardtype', 806,
-    'sp_guardsf', 807,
-    'sp_guardsfouter', 808,
-    'sp_guardobj', 809,
-    'sp_guardjustconc', 810,
-    'sp_guardjusttype', 811,
-    'sp_rebless', 812,
-    'sp_resolvecode', 813,
-    'sp_decont', 814,
-    'sp_getlex_o', 815,
-    'sp_getlex_ins', 816,
-    'sp_getlex_no', 817,
-    'sp_getarg_o', 818,
-    'sp_getarg_i', 819,
-    'sp_getarg_n', 820,
-    'sp_getarg_s', 821,
-    'sp_fastinvoke_v', 822,
-    'sp_fastinvoke_i', 823,
-    'sp_fastinvoke_n', 824,
-    'sp_fastinvoke_s', 825,
-    'sp_fastinvoke_o', 826,
-    'sp_paramnamesused', 827,
-    'sp_getspeshslot', 828,
-    'sp_findmeth', 829,
-    'sp_fastcreate', 830,
-    'sp_get_o', 831,
-    'sp_get_i64', 832,
-    'sp_get_i32', 833,
-    'sp_get_i16', 834,
-    'sp_get_i8', 835,
-    'sp_get_n', 836,
-    'sp_get_s', 837,
-    'sp_bind_o', 838,
-    'sp_bind_i64', 839,
-    'sp_bind_i32', 840,
-    'sp_bind_i16', 841,
-    'sp_bind_i8', 842,
-    'sp_bind_n', 843,
-    'sp_bind_s', 844,
-    'sp_p6oget_o', 845,
-    'sp_p6ogetvt_o', 846,
-    'sp_p6ogetvc_o', 847,
-    'sp_p6oget_i', 848,
-    'sp_p6oget_n', 849,
-    'sp_p6oget_s', 850,
-    'sp_p6obind_o', 851,
-    'sp_p6obind_i', 852,
-    'sp_p6obind_n', 853,
-    'sp_p6obind_s', 854,
-    'sp_deref_get_i64', 855,
-    'sp_deref_get_n', 856,
-    'sp_deref_bind_i64', 857,
-    'sp_deref_bind_n', 858,
-    'sp_getlexvia_o', 859,
-    'sp_getlexvia_ins', 860,
-    'sp_getstringfrom', 861,
-    'sp_getwvalfrom', 862,
-    'sp_jit_enter', 863,
-    'sp_boolify_iter', 864,
-    'sp_boolify_iter_arr', 865,
-    'sp_boolify_iter_hash', 866,
-    'sp_cas_o', 867,
-    'sp_atomicload_o', 868,
-    'sp_atomicstore_o', 869,
-    'prof_enter', 870,
-    'prof_enterspesh', 871,
-    'prof_enterinline', 872,
-    'prof_enternative', 873,
-    'prof_exit', 874,
-    'prof_allocated', 875,
-    'ctw_check', 876,
-    'coverage_log', 877,
-    'breakpoint', 878);
+    'atkey_i8', 804,
+    'atkey_i16', 805,
+    'atkey_i32', 806,
+    'atkey_u8', 807,
+    'atkey_u16', 808,
+    'atkey_u32', 809,
+    'atkey_u64', 810,
```

Comment:
Adding a `_u` is fine, but the others should not exist. Somewhere around [here](https://github.com/MoarVM/MoarVM/blob/master/src/6model/reprs/MVMContext.c#L51) it should be possible to do a `switch` over `frame->static_info->body.lexical_types[entry->value]` for all the sized types, and accept the integer ones if `kind` is `MVM_reg_in64` and so forth. Similar will be needed for handling sized num32 too, which this PR doesn't address.